src/prototype_room_memory.js

```diff
@@ -152,6 +152,93 @@ Room.prototype.getMemoryPath = function(name) {
 };
 
 /**
+ * Returns a pseudo-path including every tile adjacent to one side of the given path
```

Comment:
this function should be placed in _routing or somewhere else, but not in _memory file

Reply:
I'm conflicted. It's theoretically a routing function, but the only place it's currently called is in_memory. I decided to put it here until/unless it gets used somewhere else. I could easily be convinced otherwise.